src/app/sidenav/sidenav.component.ts

```diff
@@ -48,10 +51,13 @@ export class MzSidenavComponent implements AfterViewInit, OnDestroy {
     this.renderer.setElementAttribute(collapseButton, 'data-activates', this.id);
 
     // Initialize collapsible button for side navigation
-    $(collapseButton).sideNav({
+    $(collapseButton).sideNav(<any>{
```

Comment:
It should be cast as SidenavOptions object from typing.

Reply:
Read the comment of the PR ;)